### PR TITLE
[NEXUS 10745] upgrade-guide-images

### PR DESCRIPTION
**Pull request by dulanism**
JIRA: https://issues.sonatype.org/browse/NEXUS-10745
CI: http://bamboo.s/browse/BOOK-NEXUS192

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ echo "Building documentation index page"
 asciidoc -o target/nexus-documentation.html nexus-documentation.asciidoc
 
 echo "Building pdf version of book"
-a2x -k -fpdf -dbook --xsl-file=docbook-xsl/fo.xsl --dblatex-opts="-P toc.section.depth=2 -P latex.output.revhistory=0 -P doc.publisher.show=0 -s ./latex/custom-docbook.sty" -D target book-nexus.asciidoc
+a2x -v -k -fpdf -dbook --xsl-file=docbook-xsl/fo.xsl --dblatex-opts="-P toc.section.depth=2 -P latex.output.revhistory=0 -P doc.publisher.show=0 -s ./latex/custom-docbook.sty" -D target book-nexus.asciidoc
```
Comment (mosabua) [inline build.sh]: This is for debugging only.. please take it out for CI builds..

File: chapter-upgrading.asciidoc [hunk 2]
```diff
@@ -381,11 +385,16 @@ Agent Connection:: This screen presents two fields, 'URL' and 'Access Token'. In
...
-'NuGet API-Key', 'Realms', 'Roles', 'SSL Certificates', 'Target Privileges', 'Users', 'Crowd', and 'User Tokens'. 
-For 'System' you can select 'Email', 'HTTP Configuration', and 'IQ Server' configurations.
+Content:: This screen, shown in <<fig-upgrade-content-options>>, allows you to select from compatible component 
+formats ('Repositories'), security features ('Security'), and server configuration ('System'). For 'Repositories' 
+you can select 'User-Managed Repositories', 'Repository Targets', and 'Health Check'. For 'Security' you can 
+choose from 'Anonymous', 'LDAP Configuration', 'NuGet API-Key', 'Realms', 'Roles', 'SSL Certificates', 'Target 
+Privileges', 'Users', 'Crowd', and 'User Tokens'. For 'System' you can select 'Email', 'HTTP Configuration', and 
+'IQ Server' configurations.
+
+[[fig-upgrade-content-options]]
```
Comment (mosabua) [inline chapter-upgrading.asciidoc]: While nice to have this image will potentially be out of sync with the real features soon.. a bit of a PITA to maintain. Might be worth chopping off or mentioning that its an example.

File: chapter-upgrading.asciidoc [hunk 3]
```diff
@@ -372,7 +372,11 @@ access the 'Upgrade' tool. Follow these steps:
 After you enable the upgrade capabilities, access the upgrade tool in {nxrm} 3 to start your upgrade.
 
 . Go to the 'Administration' menu.
-. Select 'Upgrade' located under 'System' to open the wizard.
+. Select 'Upgrade' located under 'System' to open the wizard, shown in <<fig-upgrade-wizard-intro>>.
+
+[[fig-upgrade-wizard-intro]]
```
Comment (mosabua) [inline chapter-upgrading.asciidoc]: The text in this image will change quite a bit probably and imho it provides no value to have as an image.. I would toss it.

Comment (dulanism) [inline chapter-upgrading.asciidoc]: Will do.

File: chapter-upgrading.asciidoc (hunk 2 above)
Comment (dulanism) [inline chapter-upgrading.asciidoc]: Agreed. I'm fine with removing it.

**Comment by dulanism** [comment]
+1 to @mosabua's image suggestions.

**Pull request merged**